### DISCUSSION TITLE
We are using Kubernetes version 1.18 with a executable binary of kube-scheduler, how to replace the default scheduler with the scheduler-plugins image

### DISCUSSION THREAD

**Discussion by yauldmar** [kubernetes-sigs/scheduler-plugins#650] [Q&A]
We deployed kube-scheduler like this:

> 
> vim /etc/systemd/system/kube-scheduler.service
> [Unit]
> Description=Kubernetes Scheduler Documentation=https://github.com/GoogleCloudPlatform/kubernetes After=kube-apiserver.service
> Requires=kube-apiserver.service
> [Service] ExecStart=/opt/kube/bin/kube-scheduler \
> --bind-address=XX.XX.XX.XX \ 
>  --master=http://XX.XX.XX.XX:xxxx \ 
>  --leader-elect=true \
> --logtostderr=false \
> --alsologtostderr=true \ --log-dir=/var/log/kubernetes \ --v=1
> Restart=on-failure LimitNOFILE=65536
> [Install] WantedBy=multi-user.target

then
> systemctl start kube-scheduler

**Comment by Huang-Wei** [comment]
This repo's release only contain images. If you run scheduler as systemd service, you'd need to build the binary (via `make build-scheduler`), and put it in `/opt/kube/bin/kube-scheduler` and finally `sudo systemctl restart kube-scheduler`. (⚠️ ensure the original kube-scheduler binary is backed up)
